File: Modul3/SOAL4/PRAK304-2410817120013-OktaviaLailaQodari.c
#include <stdio.h>

int main() {
    int bil;
    scanf("%d", &bil);
    if (bil == 0) {
        printf("Nol\n");
    }
    else if (bil >= 1 & bil <10) {
        printf("Satuan");
    }
    else if (bil >=11 & bil < 20) {
        printf("Belasan");
    }
    else if (bil ==10 || bil >= 20 & bil <= 99) {
        printf("Puluhan");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
}

File: Modul3/SOAL4/PRAK304-2410817120013-OktaviaLailaQodari.py
bil = int(input())

if bil == 0:
    print("Nol")
elif bil >= 1 and bil < 10:
    print("Satuan")
elif bil >= 11 and bil < 20:
    print("Belasan")
elif bil ==10 or bil >= 20 and bil <= 99:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")
